installers/install.ps1

```diff
@@ -155,7 +162,7 @@ catch
 {
     progress_fail
     Write-Error $_.Exception.Message
-    return 1
+    exit 1
```

Comment:
Was this necessary? Little concerned of whatever the result of this might be. Were we just exiting zero before?

Reply:
Yes, we were exiting zero before.